apps/condo/domains/organization/integrations/sbbol/sync/syncUser.js

```diff
@@ -68,42 +80,60 @@ const cleanEmailForAlreadyExistingUserWithGivenEmail = async ({ email, userIdToE
  * @return {Promise<{importId}|*>}
  */
 const syncUser = async ({ context, userInfo }) => {
-    const returnFields = 'id phone email name importId importRemoteSystem'
-    const importFields = {
-        type: 'staff',
-        importId: userInfo.importId,
-        importRemoteSystem: userInfo.importRemoteSystem,
+    const returnFields = 'id phone email name'
+    const identityWhereStatement = {
+        user: { type: 'staff' },
```

Comment:
we have a const for that!

Reply:
Ok, I just take that code from rest of syncUser.js - I'll replace this with constant